packages/story-editor/src/components/elementLink/output.js

```diff
@@ -31,14 +31,17 @@ function WithLink({ element, children, ...rest }) {
   if (!link?.url?.length) {
     return children;
   }
+  const rel = link?.rel.join(' ') ?? '';
   const urlWithProtocol = withProtocol(link.url);
   return (
+    // False positive.
+    // eslint-disable-next-line react/jsx-no-target-blank
     <a
       href={urlWithProtocol}
       data-tooltip-icon={link.icon}
       data-tooltip-text={link.desc}
       target="_blank"
-      rel="noreferrer"
+      rel={`${rel} noreferrer`}
```

Comment:
Nit: when `rel` is empty, this will result in `rel=" noreferrer"`.

Might want to trim.